crates/parry2d/Cargo.toml

```diff
@@ -68,4 +85,95 @@ simba = { version = "0.9", default-features = false }
 oorandom = "11"
 ptree = "0.4.0"
 rand = { version = "0.8" }
-macroquad = "0.4"
\ No newline at end of file
+macroquad = "0.4"
+
+[package.metadata.docs.rs]
+rustdoc-args = ["-Zunstable-options", "--cfg", "docsrs"]
+all-features = true
+cargo-args = ["-Zunstable-options", "-Zrustdoc-scrape-examples"]
+
+[[example]]
+name = "aabb2d"
+path = "examples/aabb2d.rs"
+doc-scrape-examples = true
```

Comment:
Add a comment to explain why all examples are listed (rustdoc scrape examples)

+ check if rustc has planned to facilitate this configuration